e2e-tests/affinity/compare/statefulset_custom-proxysql-oc.yml

```diff
@@ -4,7 +4,7 @@ metadata:
   generation: 1
   name: custom-proxysql
   ownerReferences:
-  - apiVersion: pxc.percona.com/v1
+  - apiVersion: pxc.percona.com/v1-6-0
```

Comment:
please add the following command into these lines https://github.com/percona/percona-xtradb-cluster-operator/blob/master/e2e-tests/functions#L201-L216
```
egrep -v "apiVersion: pxc.percona.com"
```
and regenerate all `e2e-tests/*/compare/*.yml` files